core/handler/device/store.go

```diff
@@ -23,11 +24,13 @@ type Store interface {
 	DownlinkQueue(appID, devID string) (DownlinkQueue, error)
 	Set(new *Device, properties ...string) (err error)
 	Delete(appID, devID string) error
+	AttrWhitelist() map[string]bool
 }
 
 const defaultRedisPrefix = "handler"
 const redisDevicePrefix = "device"
 const redisDownlinkQueuePrefix = "downlink"
+const redisUpStreamMetadataKeyWhitelist = ":attr-whitelist"
```

Comment:
1) This is just configuration, and you load it on start. Why do we need to store this in Redis, instead of using the handler's config?
2) `s/redisUpStreamMetadataKey/redisDeviceAttribute/g`
3) lose the colon, that belongs in the formatting, not here